partiql-parser/src/parse/partiql.lalrpop

```diff
@@ -15,7 +15,44 @@ use partiql_source_map::location::{ByteOffset, BytePosition, ToLocated};
 grammar<'input, 'err>(input: &'input str,
                       errors: &'err mut Vec<ErrorRecovery<ByteOffset, lexer::Token<'input>, ParseError<'input, BytePosition>>>);
 
+
+// ------------------------------------------------------------------------------ //
+//                                                                                //
+//                            UNION/INTERSECT/EXCEPT                              //
+//                                                                                //
```

Comment:
Not a blocker: How about adding a note about the higher precendence of `INTERSECT`?